src/System.Text.Json/tests/Serialization/TestClasses.cs

```diff
@@ -769,6 +769,48 @@ public void Verify()
         }
     }
 
+    public class TestClassWithStringToObjectDictionaries : ITestClass
+    {
+        public Dictionary<string, SimpleTestClass> MyDict { get; set; }
+
+        // TODO: Add support for classes as dictionary values in class-nested dicitonaries for the following types.
```

Comment:
Is this just a TODO for right now? or is something blocking it?

Reply:
Yes, just a TODO. Want to get this through first - just a little cleaner in my head.